src/commands/stats.ts

```diff
@@ -34,7 +34,7 @@ export const stats: CommandDef = {
           },
           {
             name: 'Bot Online Time',
-            value: 'Wake Time from JSON file'
+            value: config.ONLINE_AT
           },
           {
             name: 'Version',
```

Comment:
A quick question here - is hard-coding the repository URL the best route? Was testing these changes, and if I've done a commit on my repo, the bot pulls that commit hash but tries to link to this repo.

Reply:
There is a way to make it link it's own repository, but only locally. If it's going to be deployed on heroku, there is no way of finding where the repository is at. 
But, making it a config item would be a great idea. Ask the user which repository they want it linked to. 
Or even, we can grab the repository from the `package.json` as a fallback, then if they didn't specify any, it will just be Brad's repository that is linked.

Reply:
We can try asking Brad, if he just wants it hard-coded, or as a config file, or getting it from the `package.json` or all of them with fallbacks.

Reply:
The simplest solution to this would be to grab the information from the `package.json` on load of the script, however the problem wont exist in production environments, only in forked repos which is to be expected.

I think it should be **ok** to leave as is, to keep complexity down for the sake of simplicity. :)

Reply:
Sounds good to me!